src/Sulu/Bundle/HttpCacheBundle/Resources/config/structure-cache-handlers.xml

```diff
@@ -3,44 +3,34 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
-    <parameters>
-        <parameter key="sulu_http_cache.handler.aggregate.class">Sulu\Component\HttpCache\Handler\AggregateHandler</parameter>
-        <parameter key="sulu_http_cache.handler.tags.class">Sulu\Component\HttpCache\Handler\TagsHandler</parameter>
-        <parameter key="sulu_http_cache.handler.paths.class">Sulu\Component\HttpCache\Handler\PathsHandler</parameter>
-        <parameter key="sulu_http_cache.handler.public.class">Sulu\Component\HttpCache\Handler\PublicHandler</parameter>
-        <parameter key="sulu_http_cache.handler.debug.class">Sulu\Component\HttpCache\Handler\DebugHandler</parameter>
-    </parameters>
```

Comment:
I think that's ok in this case, but actually we only don't use parameters for the services we add. Removing these parameters is a BC break.